app/views/stages/_fields.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
       <div class="col-lg-4">
         <%= form.collection_select(
```

Comment:
might be simpler to move the lookup here so we don't have to repeat logic in 2 placed + break when that logic gets out of sync